root/lib/systemd/system/titus-salt-generator.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Write salt for Titus IMDS
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+ExecStart=/bin/bash -cx 'echo -n TOKEN_KEY_SALT=\" > /run/titus-metadata-proxy.salt && /bin/cat /proc/sys/kernel/random/uuid | tr -d "\n" >> /run/titus-metdata-proxy.salt && echo \" >> /run/titus-metdata-proxy.salt'
```

Comment:
Maybe I'm missing something, but why do we need the quotes around setting the variable, and the tr?

Reply:
I guess it's technically not needed. I quote out of habit and caution (less potential for breakage). `tr` is necessary since `/proc/sys/kernel/random/uuid` outputs a newline.

Reply:
It became an escaping nightmare. I ended up reverting to what you wrote.